docs/package.json

```diff
@@ -3,13 +3,14 @@
   "name": "@aws-amplify/ui-docs",
   "version": "0.0.1",
   "scripts": {
-    "dev": "next dev -p 5001",
+    "dev": "yarn prebuild && yarn next dev -p 5001",
     "preflutter:build": "yarn flutter:clean",
     "flutter:build": "(cd flutter/authenticator && flutter build web)",
     "postflutter:build": "yarn flutter:copy",
     "flutter:clean": "rm -rf public/flutter/authenticator",
     "flutter:copy": "cp -r ./flutter/authenticator/build/web public/flutter/authenticator",
-    "build": "yarn build:cssvars-table && yarn build:props-tables && NODE_OPTIONS=\"--max-old-space-size=4096\" next build",
+    "prebuild": "yarn build:cssvars-table && yarn build:props-tables",
+    "build": "NODE_OPTIONS=\"--max-old-space-size=4096\" next build",
```

Comment:
Will hosting still build the props table with this change? Or is hosting configured to run prebuild also?

Reply:
prebuild should always run before the build command

Reply:
Each script can have a `pre` and `post` hook which will run accordingly (pre will run before the script, while post will run after the script). You can read more here: https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts